impl/src/test/java/org/ehcache/docs/GettingStarted.java

```diff
@@ -147,7 +151,6 @@ public void threeTiersCacheManager() throws Exception {
   public void defaultSerializers() throws Exception {
     // tag::defaultSerializers[]
     CacheConfiguration<Long, String> cacheConfiguration = CacheConfigurationBuilder.newCacheConfigurationBuilder()
-        .add(new OnHeapStoreServiceConfiguration().storeByValue(true)) // <1>
```

Comment:
You cannot just remove lines that have a callout, the corresponding entry in doc must be adapted and subsequent numbers too.